VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -51,6 +52,17 @@ def get_loop_devices(self):
             if 'loop' in file_system_info[0]:
                 disk_loop_devices_file_systems.append(file_system_info[0])
         return disk_loop_devices_file_systems
+  
+    def device_list_for_billing(self):
+        self.logger.log("In device_list_for_billing",True)
+        devices_to_bill = [] #list to store device names to be billed
+        device_items = disk_util.get_device_items(None)
+        for device_item in device_items :
+            # self.logger.log("Device name : {0} ".format(str(device_item.name)),True)
+            if str(device_item.name).startswith("sd"):
```

Comment:
check for 32 disks

Reply:
if tested for 32 disks. @mearvind  please mark it resolved

Reply:
I asked for it from Arpit but did not get it till now